src/wp-includes/class-wp-scripts.php

```diff
@@ -464,7 +533,19 @@ public function print_inline_script( $handle, $position = 'after', $display = tr
 		$output = trim( implode( "\n", $output ), "\n" );
 
 		if ( $display ) {
-			printf( "<script%s id='%s-js-%s'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), esc_attr( $position ), $output );
+			if ( 'after-non-standalone' === $position ) {
```

Comment:
@spacedmonkey I updated this based on your prior suggestion around cleaning this up, with the exception that I had to assign the script string to a var other than `$output` as that resulted in unexpected output given that we used that as the string to `sprintf` on, as well as a replacement therein.